CONTRIBUTING.md

```diff
@@ -93,26 +75,25 @@ Now you can build and run BotKube by one of the following ways
 
 ## Making A Change
 
-* *Before making any significant changes, please [open an
-issue](https://github.com/infracloudio/botkube/issues).* Discussing
-your proposed changes ahead of time will make the contribution process
-smooth for everyone.
+* Before making any significant changes, please [open an issue](https://github.com/infracloudio/botkube/issues).
+
+* Discussing your proposed changes ahead of time will make the contribution process smooth for everyone.
```

Comment:
I think these two should be one point. What do you think?

Reply:
Sounds good